chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+
+% Data set
+The most important output of data cleaning is the cleaned data set. 
+It should contain the same information as the raw data set,
+with no changes to data points.
+It should also be easily traced back to the survey instrument,
+and be accompanied by a dictionary or codebook.
+Typically, one cleaned data set will be created for each data source,
+i.e. per survey instrument.
+Each row in the cleaned data set represents one survey entry or unit of observation.\sidenote{\cite{tidy-data}}
+If the raw data set is very large, or the survey instrument is very complex,
+you may want to break the data cleaning into sub-steps, 
+and create intermediate cleaned data sets
+(for example, one per survey module).
+Breaking cleaned data sets into the smallest unit of observation inside a roster
+make the cleaning faster and the data easier to handle during construction.
+But having a single cleaned data set will help you with sharing and publishing the data.
+To make sure this file doesn't get too big to be handled,
+use commands such as \texttt{compress} in Stata to make sure the data
+is always stored in the most efficient format.
+
+% Documentation
+Throughout the data cleaning process, you will need inputs from the field, 
+including enumerator manuals, survey instruments, 
+supervisor notes, and data quality monitoring reports.
+These materials are part of what we call data documentation
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Documentation}}
+\index{Documentation},
+and should be stored in the corresponding folder, 
+as you will probably need them during analysis and publication.
+Include in the \texttt{Documentation} folder records of any
+corrections made to the data, including to duplicated entries,
+as well as communications from the field where theses issues are reported.
+Make sure to also have a record of potentially problematic patterns you noticed
+while exploring the data, such as outliers and variables with many missing values.
+Be very careful not to include sensitive information in 
+documentation that is not securely stored, 
+or that you intend to release as part of a replication package or data publication.
+
+\section{Indicator construction}
 
+% What is construction -------------------------------------
+Any changes to the original data set happen during construction.
+It is at this stage that the raw data is transformed into analysis data.
+This is done by creating derived variables
+(binaries, indices, and interactions, to name a few).
+To understand why construction is necessary,
+let's take the example of a household survey's consumption module.
+It will result in separate variables indicating the 
+amount of each item in the bundle that was consumed.
+There may be variables indicating the cost of these items.
+You cannot run a meaningful regression on these variables. 
+You need to manipulate them into something that has \textit{economic} meaning. 
```

Comment:
This example is a bit confusing. I am all in favour of specific examples to help with intuition, but this is half-specific as we make the specific example of a consumption module, and make non specific comments of what goes into a consumption module. Let me know if you want my help to re-think this example.

Reply:
how about 092fe91?